### PR TITLE
Chat merge for migration 

### PR DESCRIPTION
**Pull request by YOGURSIO**

### CODE REVIEW

File: client/src/app/pages/messages/messages.component.html [hunk 1]
```diff
@@ -1 +1,13 @@
-<p>messages works!</p>
+<app-top-navbar [leftItem]="'logo'" [pageTitle]="'Chats'"></app-top-navbar>
+<div class="top-navbar-bottom-navbar-exclusive">
```
Comment (alfonsoLeandro) [inline client/src/app/pages/messages/messages.component.html:2]: Well done using the new classes

File: client/src/app/pages/messages/messages.component.html [hunk 2]
```diff
@@ -1 +1,13 @@
...
+    <div *ngFor="let chat of chats" [routerLink]="['/chat', chat.id]" class="chat-card">
```
Comment (alfonsoLeandro) [inline client/src/app/pages/messages/messages.component.html:3]: You could have used a `<ul>`, or a `<mat-list>` here, you should consider doing so after the migration

Comment (YOGURSIO) [reply client/src/app/pages/messages/messages.component.html:3]: You mean instead of a div? I was using one before but it gave some css problems.

Comment (alfonsoLeandro) [reply client/src/app/pages/messages/messages.component.html:3]: Yes, we should check those out, weird

**Review by alfonsoLeandro — APPROVED** [review]
LGMT for now

**Review by danacastroman — APPROVED** [review]
LGTM

**Pull request merged**